tests/test_calc_barycentric_corr.py

```diff
@@ -233,6 +241,114 @@ class Params(object):
         # fmt: on
         self.assertTrue(self._check_barycentric(baryCoeff, baryExpected))
 
+    def test_single_image_barycentric(self):
```

Comment:
this test doesn't actually test for the correctness of the correction, would it be possible to add a check to make sure the corrected values make sense?

Reply:
urp. I should have removed that test! Will do so now.